### DISCUSSION TITLE
Challenges of moving a wordpress website to nextjs

### DISCUSSION THREAD

**Discussion by Boytobeaman** [WebDevStudios/nextjs-wordpress-starter#830] [Q&A]
We know nextjs is good technically, I now want to moving my wordpress website about [vegetable crates](https://www.vegcrates.com/) to nextjs. But there are a few challenges

### How to maintaine the contents for the marketing person

**Comment by itsamoreh** [comment]
Hi @Boytobeaman! Thanks for taking a look at the starter.

> How to handle the ACF fields in wordpress

For this you'll need to install [`wp-graphql-acf`](https://github.com/wp-graphql/wp-graphql-acf). This plugin will expose ACF field data to the WPGraphQL Schema. Check out their README for more details.

> How to maintaine the contents for the marketing person

Our starter supports core Gutenberg blocks out of the box. Your marketing person should be able to edit pages using Gutenberg.

If you don't want to use Gutenberg, you'll have to disable it. Your marketing person should be able to edit pages using the classic editor and everything should still work.
